src/styles/index.css

```diff
@@ -0,0 +1,418 @@
+  background: var(--image);
+}
+
+.cover-btn-container {
+  display: flex;
+  flex-direction: row;
+  gap: 0.75rem;
+  margin: 1rem auto 4rem;
+}
+
+.cover-btn {
+  height: 1rem;
+  width: 2.25rem;
+  background: var(--image);
+  border-radius: 0.25rem;
+  border: none;
+}
+
+.active-cover-btn {
+  background: var(--active-cover-btn);
+}
+
+.secondary-image {
+  aspect-ratio: 1.97 / 1;
+  border-radius: 0.5rem;
+  margin-bottom: 1.5rem;
+  background: var(--image);
+}
+
+.newsletter-heading {
+  font: 1rem "Montserrat", sans-serif;
+  text-align: center;
+  margin-bottom: 1rem;
+  color: var(--heading);
+}
+
+.newsletter-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5rem;
+  padding-bottom: 3rem;
+  background: var(--light-blue);
+}
+
+.newsletter-paragraph {
+  max-width: 20rem;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  text-align: center;
+  margin-bottom: 1.5rem;
+}
+
+.newsletter-box {
+  display: flex;
+  height: 2.5rem;
+  width: 20.375rem;
+  max-width: 21rem;
+}
+
+.newsletter-input {
+  border: none;
+  border-radius: 0.375rem 0 0 0.375rem;
+  flex-grow: 1;
+  padding: 1rem;
+  background: var(--background);
+}
+
+.newsletter-input:focus {
+  outline: none;
+}
+
+.newsletter-input::placeholder {
+  font: 0.625rem "Lato", sans-serif;
+}
+
+.newsletter-button {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+  align-items: center;
+  border: none;
+  border-radius: 0 0.375rem 0.375rem 0;
+  width: 6.125rem;
+  background: var(--subscribe-button);
+  cursor: pointer;
+}
+
+.newsletter-subscribe {
+  font: 0.625rem "Lato", sans-serif;
+  color: var(--subscribe-text);
+  padding-left: 1rem;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2.25rem 0 8rem;
+  background: #152b79;
+}
+
+.social-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 1.5rem;
+  list-style: none;
+}
+
+.author {
+  color: var(--light-blue);
+  margin: 1rem 0;
+  font-size: 1.25rem;
+  font: 0.875rem "Lato", sans-serif;
+}
+
+.footer-links {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 1rem;
+}
+
+.footer-link {
+  font: 0.625rem "Lato", sans-serif;
+  color: var(--footer-links);
+  text-decoration: underline;
+}
+
+/* Tablet dimensions */
+@media only screen and (min-width: 834px) {
+  nav {
+    justify-content: center;
+    gap: 32.5rem;
+    padding: 0 4rem;
+  }
+
+  .main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 3rem 4rem;
+  }
+
+  .primary-section,
+  .secondary-section {
+    margin-bottom: 5rem;
+  }
+
+  .primary-section .section-heading {
+    font-size: 1.375rem;
+  }
+
+  .section-content {
+    max-width: 21.77rem;
+  }
+
+  .primary-section .section-content {
+    max-width: 29.25rem;
+  }
+
+  .content-seperator {
+    margin: 0.5rem 0 0.5rem;
+  }
+
+  .primary-section .content-seperator {
+    width: 9rem;
+  }
```

Comment:
You could use `::after` pseudo element to do make this styling instead of adding more tags to HTML.